comet/utils/formatting.py

```
@@ -12,7 +13,9 @@ def normalize_info_hash(info_hash: str) -> str:
                 "utf-8"
             )
         except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)